pages/research_software_stories/acts_research_software_story.md

```diff
@@ -0,0 +1,177 @@
+  finding, etc.
+
+- Event Data Model (EDM): A standardised way to represent track data.
+
+ACTS is optimised for speed, using compile-time polymorphism (templates)
+instead of runtime inheritance to improve performance.  It keeps
+non-optional dependencies to a minimum (mainly the Eigen linear algebra
+library).  The toolkit uses CMake for building and runs on Linux with
+multi-core support.  It can integrate with external geometry definitions
+(eg.  ROOT TGeo or DD4hep), making it easier to use in different
+experiments' software frameworks.
+
+
+## Libraries and Systems
+
+- C++20: Leverages modern C++ features for performance and concurrency.
+
+- Eigen: A linear algebra library used for efficient mathematical
+  computations.
+
+- Geometry converters: Interfaces to import detector geometry from ROOT TGeo
+  and DD4hep.
+
+- GPU acceleration: Experimental support for using GPUs to speed up certain
+  computations.
+
+- ML integration: Plugins like Exa.TrkX for GNN based tracking or ONNX for
+  development and testing of ML algorithms for ambiguity resolution, seed
+  filtering etc.
+
+
+## Software Practices
+
+ACTS development follows strict software engineering standards.  The project
+is developed openly on GitHub, and every code submission triggers automated
+CI tests to ensure nothing breaks and that thread-safety is maintained. 
+Developers adhere to coding and contribution guidelines, and use peer
+reviews on merge requests.  Issues are tracked transparently, and frequent
+meetings plus mailing list discussions keep contributors aligned and
+informed.
+
+
+## Community
+
+The ACTS community provides many resources to help onboard newcomers:
+
+- Contribution guidelines: A clear step-by-step guide for submitting code.
+
+- Interactive tutorials: Hands-on guides for configuring and running
+  tracking workflows.
+
+- Example framework: A sample project demonstrating how to use ACTS
+  algorithms in practice.
+
+In addition, regular workshops and training sessions help new members start
+using ACTS in their experiments.  New contributors are often mentored by
+experienced developers, ensuring knowledge transfer and continuous growth of
+the community.
+
+
+## Tools
+
+ACTS uses common tools to maintain quality and facilitate contributions:
+
+- Version control: Git, with repositories on GitHub for collaborative
+  development.
+
+- Continuous integration: Automated testing pipelines using GitHub Actions
+
+- Documentation: Documentation written with Sphinx and published on
+  ReadTheDocs.
+
+- Build system: CMake for consistent compilation across platforms.
+
+- Issue tracking: GitHub issue trackers for transparent bug reports and
+  feature requests.
+
+- Forums: Github Discussions, Mattermost messaging and e-mail lists for open
+  communication with the core development team.
+
+These tools streamline development, ensure reliability, and make it easier
+for new users to adopt and contribute to ACTS.
+
```

Comment:
This is just a comment - and perhaps something we need to bring up with the ETT folk - lots of tools are mentioned here - and it's unfortunate that we can't use the {% tool "<tool name from tool file>" %} syntax to create tool tips - perhaps this is something we need to ask Bert - i.e. can we do this on arbitrary sections or is it special magic just for the task pages.